examples/zoom-control-directive/index.ts

```diff
@@ -2,133 +2,100 @@
 import 'reflect-metadata';
 import 'zone.js';
 
-import {YagaModule, MapComponent, ControlPosition} from '../../lib/index'; // @yaga/leflet-ng2
+import { YagaModule } from '../../lib/index'; // @yaga/leflet-ng2
 
-import {Component, AfterViewInit, ViewChild, PlatformRef} from '@angular/core';
-import {NgModule}      from '@angular/core';
-import {BrowserModule} from '@angular/platform-browser';
-import {FormsModule}   from '@angular/forms';
-import {platformBrowserDynamic} from '@angular/platform-browser-dynamic';
+import { Component, PlatformRef } from '@angular/core';
+import { NgModule }      from '@angular/core';
+import { BrowserModule } from '@angular/platform-browser';
+import { FormsModule }   from '@angular/forms';
+import { platformBrowserDynamic } from '@angular/platform-browser-dynamic';
 
+import { ExampleAppComponentBlueprint, IExampleProperties } from '../app-component-blueprint';
+import { ExamplePropertiesModule, PROPERTIES_WRAPPER } from '../property.component';
 
 const platform: PlatformRef = platformBrowserDynamic();
 
 /* tslint:disable:max-line-length */
 const template: string = `
-
-<div class="container">
-    <h3>Zoom-Control Example</h3>
-</div>
-
+<example-header [title]="'Zoom-Control-Directive'"></example-header>
 <div class="container">
   <div class="map">
     <yaga-map>
+      <yaga-zoom-control
       
-      <yaga-tile-layer [url]="'http://a.tile.openstreetmap.org/{z}/{x}/{y}.png'">
-      </yaga-tile-layer>
-      
-      <yaga-zoom-control 
-         [(position)]="position"
-         [zoomInText]="zoomInText"
-         [zoomInTitle]="zoomInTitle"
-         [zoomOutText]="zoomOutText"
-         [zoomOutTitle]="zoomOutTitle"
-      >
+        (click)="handleEvent('click', $event);"
+        (dblclick)="handleEvent('dblclick', event);"
+        (mousedown)="handleEvent('mousedown', event);"
+        (mouseup)="handleEvent('mouseup', event);"
+        (mouseover)="handleEvent('mouseover', event);"
+        (mouseout)="handleEvent('mouseout', event);"
+        (mousemove)="handleEvent('mousemove', event);"
+        (positionChange)="handleEvent('positionChange', event);"
+        (displayChange)="handleEvent('displayChange', event);"
+
+        [zoomInText]="getInputPropertyByName('zoomInText').value"
+        [zoomInTitle]="getInputPropertyByName('zoomInTitle').value"
+        [zoomOutText]="getInputPropertyByName('zoomOutText').value"
+        [zoomOutTitle]="getInputPropertyByName('zoomOutTitle').value"
+        
+        [position]="getDuplexPropertyByName('position').value"
+        [(zIndex)]="getDuplexPropertyByName('zIndex').value"
+        [(display)]="getDuplexPropertyByName('display').value"
+        [opacity]="getDuplexPropertyByName('opacity').value"
```

Comment:
Bind duplex properties in duplex mode

Here a little mule-bridge: `[(duplex)]`, `[input]`, `(output)`